lib/server.js

```diff
@@ -210,8 +212,42 @@ module.exports = {
       const parts = path.split(' ')
 
       let methods = []
-      if (route.config && route.config.pre && route.config.pre.length > 0) {
-        methods = methods.concat(route.config.pre)
+      if (route.config) {
+        // add validation
```

Comment:
Maybe all this stuff can move into `utils.js` to just retrieve the validate middleware, like this it will be more clear